%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% "ModernCV" CV and Cover Letter
% LaTeX Template
% Version 1.11 (19/6/14)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Xavier Danaux (xdanaux@gmail.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
% Important note:
% This template requires the moderncv.cls and .sty files to be in the same 
% directory as this .tex file. These files provide the resume style and themes 
% used for structuring the document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[11pt,a4paper,sans]{moderncv} % Font sizes: 10, 11, or 12; paper sizes: a4paper, letterpaper, a5paper, legalpaper, executivepaper or landscape; font families: sans or roman

\moderncvstyle{casual} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{black} % CV color - options include: 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'

\usepackage{lipsum} % Used for inserting dummy 'Lorem ipsum' text into the template
\usepackage[scale=0.75]{geometry} % Reduce document margins


\usepackage[T1]{fontenc}
\usepackage{lmodern}
%\setlength{\hintscolumnwidth}{3cm} % Uncomment to change the width of the dates column
%\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name

%----------------------------------------------------------------------------------------
%	NAME AND CONTACT INFORMATION SECTION
%----------------------------------------------------------------------------------------
\newcommand\Colorhref[3][cyan]{\href{#2}{\small\color{#1}#3}}

\firstname{Mitchell Robert} % Your first name
\familyname{Vollger} % Your last name

% All information in this block is optional, comment out any lines you don't need
\title{Curriculum Vitae}
%\address{123 Broadway}{City, State 12345}
%\mobile{(000) 111 1111}
%\phone{(000) 111 1112}
%\fax{(000) 111 1113}
%\email{john@smith.com}
%\homepage{staff.org.edu/~jsmith}{staff.org.edu/$\sim$jsmith} % The first argument is the url for the clickable link, the second argument %is the url displayed in the template - this allows special characters to be displayed such as the tilde in this example

%\extrainfo{additional information}

%\photo[70pt][0.4pt]{pictures/picture} % The first bracket is the picture height, the second is the thickness of the frame around the picture (0pt for no frame)

%\quote{"A witty and playful quotation" - John Smith}

%----------------------------------------------------------------------------------------

\begin{document}
\makecvtitle % Print the CV title
\section{\textbf{Contact Information}}
\subsection{\textbf{Email}}
\hspace{25 mm}\cvitem{Primary}{\hspace{4 mm}mvollger@uw.com}
\hspace{25 mm}\cvitem{Secondary}{\hspace{4 mm}mrvollger@gmail.com}

\subsection{\textbf{Phone}}
\hspace{25 mm}\cvitem{Mobile}{\hspace{4 mm}(707) 407-8732}
\subsection{\textbf{Address}}
\hspace{25 mm}\cvitem{Primary}{\hspace{4 mm}219 E Garfield Street Apt 217, Seattle Washington, 98102}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------



\vspace{3 mm}
\section{\textbf{Education}}

\subsection{\textbf{Degree Programs}}

%\texttt{\href{http://www.google.com}}

\cventry{2016--Present}{Ph.D Candidate in Genome Sciences}{University of Washington}{Seattle, Washington}{GPA -- 3.86}{}  % Arguments not required can be left empty


\cventry{2011--2015}{B.S.E. in Computer Science Engineering}{Princeton University}{Princeton, New Jersey}{\textit{GPA -- 3.13}. Student of the \textcolor{blue}{\small{\href{http://www.princeton.edu/integratedscience/}{Integrated Science Curriculum}}}}{}  % Arguments not required can be left empty
 \cventry{2011--2015}{Certificate in \textcolor{blue}{\href{http://www.princeton.edu/integratedscience/certificate/}{Quantitative and Computational Biology}}}{Princeton University}{Princeton, New Jersey}{\textit{GPA -- 3.48}}{}  % Arguments not required can be left empty
\cventry{2008--2011}{Associate of Arts Degree in Mathematics}{College of the Redwoods}{Eureka, California}{\textit{GPA -- 4.00}}{}
\cventry{2008--2011}{Associate of Arts Degree in Science}{College of the Redwoods}{Eureka, California}{\textit{GPA -- 4.00}}{}


%\cventry{2007--2010}{Bachelor of Business Studies}{The University of California}{Berkeley}{\textit{GPA -- 7.5}}{Specialized in Commerce}


%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{Research Experience and Independent Work}}



\vspace{3 mm}
%\subsection{\textbf{Independent Work and }}

\cventry{Spring 2018 - Present}{Doctoral Candidate}{Resolving Duplications in Genome Assembly}{University of Washington}{}{Advisor Evan Eichler, Genome Sciences. Developing and applying methods that use paralog specific variants (PSVs) to resolve duplications and improve genome assembly. }

\cventry{Spring 2017 - Spring 2018}{Predoctoral Candidate}{See previous item}{University of Washington}{}{}

\cventry{Winter 2017}{Predoctoral Candidate}{ Identifying Insertion/Deletion Events in Mendelian Diseases}{University of Washington}{}{Advisor Debbie Nickerson, Genome Sciences. Implemented a variety of variant callers on a large number of genomes across many Mendelian Diseases in order to more consistently identify insertion and deletion events}

\cventry{Fall 2016}{Predoctoral Candidate}{Tandem Identification of Multiple Charge States in MS}{University of Washington}{}{Advisor William Noble, Genome Sciences.  Implemented group LASSO to confirm the existence of a single peptide in multiple charge states in mass spectrometry data for use in data independent acquisition (DIA) deconvolution. }

\cventry{Fall 2014 - Summer 2015}{Undergraduate Researcher}{Developing a Reference Genome for W303}{Princeton University}{}{Advisor Alison Gammie, Molecular Biology Department. Developed methods to create a reference genome for W303 \textit{Saccharomyces cerevisiae} using existing high-throughput sequencing data. Relevant languages: Python, C, Java, R, and Matlab.}

\cventry{Fall 2014 - Summer 2015}{Undergraduate Researcher}{Quantifying Mutations Due to Cisplatin and UV}{Princeton University}{}{Advisor Alison Gammie, Molecular Biology Department. Developed computational methods to quantify DNA damage done to \textit{Saccharomyces cerevisiae} by UV and Cisplatin \textit{in vivo}. Relevant languages: Python, C, Java, R, and Matlab.}

\cventry{2014}{iOS Software Developer}{GeoTasker}{Princeton University}{}{Created \textcolor{blue}{\small{\href{https://sites.google.com/site/geotasker333/home}{GeoTasker}}}, a location-based reminder app released on iOS 8 in the fall of 2014, developed under the instruction of Brian Kernighan.  Relevant languages: Objective-C, and C}


\cventry{Fall 2013}{Undergraduate Researcher}{Analysis of an Artificial Transcription Factor}{Princeton University}{}{Advisors Megan McClean, Alison Gammie, Marcus Noyes. Analyzed the transcriptomes of \textit{Saccharomyces cerevisiae} induced by Msn2 and by an artificial transcription factor mimicking Msn2.  Relevant languages: R}


%\subsection{Teaching}


%----------------------------------------------------------------------------------------
%	GRANTS SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{Fellowships and Awards}}
\cvitem{Fall 2017 - Fall 2019}{BDGN, Big Data in Genomics and Neuroscience. Awarded for two years.}

\cvitem{Fall 2016 - Fall 2017}{NIH/NHGRI T32, through the Genome Training Grant. Awarded for two years.}
\cvitem{2011}{Graduated with Highest Honors, College of the Redwoods}
\cvitem{2011}{National Hispanic Recognition Program Scholar}
\cvitem{2011}{National Merit Scholarship Semifinalist}
\cvitem{2011}{Valedictorian, Academy of the Redwoods}

%----------------------------------------------------------------------------------------
%	Publications SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{Publications}}


\cvitem{\textbf{---}}{ Shafin K, Pesout T, Lorig-Roach R, Haukness M, Olsen HE, Bosworth C, et al. Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes. Nat Biotechnol. 2020. doi:10.1038/s41587-020-0503-6 }

\cvitem{\textbf{---}}{Nurk S, Walenz BP, Rhie A, \textbf{Vollger MR}, Logsdon GA, Grothe R, et al. HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads. bioRxiv. 2020. doi:10.1101/2020.03.14.992248}

\cvitem{\textbf{---}}{\textbf{Vollger MR}, Logsdon GA, Audano PA, Sulovari A, Porubsky D, Peluso P, et al. Improved assembly and variant detection of a haploid human genome using single-molecule, high-fidelity long reads. Ann Hum Genet. 2020;84: 125â€“140. doi:10.1111/ahg.12364}

\cvitem{\textbf{---}}{Porubsky D, Ebert P, Audano PA, \textbf{Vollger MR}, Harvey WT, Munson KM, et al. A fully phased accurate assembly of an individual human genome. bioRxiv. 2019. p. 855049. doi:10.1101/855049}

\cvitem{\textbf{---}}{Sulovari A, Li R, Audano PA, Porubsky D, \textbf{Vollger MR}, Logsdon GA, et al. Human-specific tandem repeat expansion and differential gene expression during primate evolution. Proc Natl Acad Sci U S A. 2019; 201912175. doi:10.1073/pnas.1912175116}

\cvitem{\textbf{---}}{ Hsieh P, \textbf{Vollger MR}, Dang V, Porubsky D, Baker C, Cantsilieris S, Hoekzema K, Lewis AP, Munson KM, Sorensen M, Kronenberg ZN, Murali S, Nelson BJ, Chiatante G, Maggiolini FAM, Blanche H, Underwood JG, Antonacci F, Deleuze J-F, Eichler EE. 2019. Adaptive archaic introgression of copy number variants and the discovery of previously unknown human genes. Science 366. doi:10.1126/science.aax2083 }

\cvitem{\textbf{---}}{  Miga KH, Koren S, Rhie A, \textbf{Vollger MR}, Gershman A, Bzikadze A, Brooks S, Howe E, Porubsky D, Logsdon GA, Schneider VA, Potapova T, Wood J, Chow W, Armstrong J, Fredrickson J, Pak E, Tigyi K, Kremitzki M, Markovic C, Maduro V, Dutra A, Bouffard GG, Chang AM, Hansen NF, Thibaud-Nissen F, Schmitt AD, Belton J-M, Selvaraj S, Dennis MY, Soto DC, Sahasrabudhe R, Kaya G, Quick J, Loman NJ, Holmes N, Loose M, Surti U, Risques RA, Graves Lindsay TA, Fulton R, Hall I, Paten B, Howe K, Timp W, Young A, Mullikin JC, Pevzner PA, Gerton JL, Sullivan BA, Eichler EE, Phillippy AM. 2019. Telomere-to-telomere assembly of a complete human X chromosome. bioRxiv. doi:10.1101/735928 }

\cvitem{\textbf{---}}{  Maggiolini FAM, Cantsilieris S, D Addabbo P, Manganelli M, Coe BP, Dumont BL, Sanders AD, Pang AWC, \textbf{Vollger MR}, Palumbo O, Palumbo P, Accadia M, Carella M, Eichler EE, Antonacci F. 2019. Genomic inversions and GOLGA core duplicons underlie disease instability at the 15q25 locus. PLoS Genet 15:e1008075. doi:10.1371/journal.pgen.1008075 }

\cvitem{\textbf{---}}{  \textbf{Vollger MR}, Dishuck PC, Sorensen M, Welch AE, Dang V, Dougherty ML, Graves-Lindsay TA, Wilson RK, Chaisson MJP, Eichler EE. 2019. Long-read sequence and assembly of segmental duplications. Nat Methods. doi:10.1038/s41592-018-0236-3 }







%----------------------------------------------------------------------------------------
%	Posters SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{Posters}}

\cventry{December 2019}{Improved Assembly of Human Genomes Using HiFi}{Annual Scientific Meeting}{Howard Hughes Medical Institute}{}{Presented a poster on my research on assembly of human genomes using accurate long reads.}

\cventry{October 2018}{Resolving segmental duplications using long reads and correlation clustering}{Collaborative Seminar Series}{Allen Institute, Fred Hutch, and UW Medicine}{}{Presented a poster on my thesis research on developing and applying methods that use paralog specific variants (PSVs) to resolve collapsed duplications to improve genome assembly. }

\cventry{September 2017/2018}{Resolving Segmental Duplications with PSV based Community Detection}{Genome Sciences Annual Retreat}{Washington University}{}{Presented a poster on my thesis research on developing and applying methods that use paralog specific variants (PSVs) to resolve collapsed duplications to improve genome assembly.  }


\cventry{April 2017}{Identifying Multiple Charge States of Peptides in Mass Spectrometry}{2017 NHGRI Annual Meeting}{Washington University in St. Louis}{}{Presented a poster on the research I did with William Noble. A description of the research can be found in the Independent Work and Research section. }


%----------------------------------------------------------------------------------------
%	Invited Talks SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{Presentations}}

\cventry{Sep 2019}{Plenary talk}{Pacific Biosciences User Group Meeting}{University of Delaware}{}{Improved Assembly of Segmental Duplications Using HiFi}


\cventry{Feb 2015}{Speaker}{The Princeton High Throughput Sequencing Group}{Princeton University}{}{Invited to give a talk on the research that I did with Alison Gammie. The research is described in the Independent Work and Research section above. }




%----------------------------------------------------------------------------------------
%	COMPUTER SKILLS SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{Computer Skills}}

\cvitem{Basic}{x86 Assembly, Objective C}
\cvitem{Intermediate}{\LaTeX, Java, C, C++}
\cvitem{Advanced}{Python, R}

%----------------------------------------------------------------------------------------
%	References SECTION
%----------------------------------------------------------------------------------------

\vspace{3 mm}
\section{\textbf{References}}

\cvitem{ \textcolor{blue}{\href{https://eichlerlab.gs.washington.edu/index.html}{E. Eichler}}}{eee@gs.washington.edu  }
\cvitem{ \textcolor{blue}{\href{http://infotheory.ee.washington.edu/}{S. Kannan }}}{ksreeram@uw.edu    }
\cvitem{ \textcolor{blue}{\href{http://chaissonlab.usc.edu/}{M. Chaisson}}}{mchaisso@usc.edu}

%\cvitem{ \textcolor{blue}{\href{http://molbio.princeton.edu/faculty/molbio-faculty/108-gammie}{A. Gammie}}}{agammie@princeton.edu  }
%\cvitem{ \textcolor{blue}{\href{https://www.cs.princeton.edu/~bwk/}{B. Kernighan }}}{bwk.cs@princeton.edu   }
%\cvitem{ \textcolor{blue}{\href{http://directory.engr.wisc.edu/bme/Faculty/Mcclean_Megan/}{M. McClean}}}{mmcclean@princeton.edu}





%----------------------------------------------------------------------------------------
%	COVER LETTER
%----------------------------------------------------------------------------------------

% To remove the cover letter, comment out this entire block

%\clearpage

%\recipient{HR Department}{Corporation\\123 Pleasant Lane\\12345 City, State} % Letter recipient
%\date{\today} % Letter date
%\opening{Dear Sir or Madam,} % Opening greeting
%\closing{Sincerely yours,} % Closing phrase
%\enclosure[Attached]{curriculum vit\ae{}} % List of enclosed documents

%\makelettertitle % Print letter title

%\lipsum[1-3] % Dummy text

%\makeletterclosing % Print letter signature

%----------------------------------------------------------------------------------------

\end{document}